--- a/opt/influx/autostart.sh	2018-03-09 14:34:56.000000000 +0200
+++ b/opt/influx/autostart.sh	2026-03-20 17:02:34.456019736 +0200
@@ -13,10 +13,7 @@
 # Avoiding the sleep during OTA update 
 INF_UPG=$(/usr/bin/fw_printenv influx_upgrade | cut -d = -f 2)
 if [ "$INF_UPG" == "1" ]; then
-	/opt/influx/rexgen_sn_to_hostname.sh	
-	/usr/bin/systemctl restart avahi-daemon.service
-	/opt/influx/check_taiscale_status.sh
-	/usr/bin/systemctl start end_influx_upgrade.service &
+	/opt/influx/mender_update_finish.sh
 fi
 
 # Power up the Quectel chip
