--- a/opt/influx/autostart.sh  2025-10-07 08:31:05.570329462 +0000
+++ b/opt/influx/autostart.sh  2025-10-07 14:03:50.983223606 +0000
@@ -4,6 +4,16 @@
 # execute at first time
 /etc/profile.d/enable_services.sh
 
+# Mender image update need this
+# remove/add this crontab jobs
+/usr/bin/crontab -u root -l |  grep -v "* /opt/influx/release_check.sh" |  /usr/bin/crontab -
+#
+# extract preserved files
+if test -f /data/mender/preserved-files.tgz; then
+    tar -xzf /data/mender/preserved-files.tgz -C /
+    mv /data/mender/preserved-files.tgz /data/mender/preserved-files_old.tgz
+fi
+
 # Power up the Quectel chip
 /opt/influx/cellular_module_start.sh
 
