--- a/opt/influx/autostart.sh	2025-11-25 13:21:10.245182611 +0000
+++ b/opt/influx/autostart.sh	2025-11-25 13:42:50.231332682 +0000
@@ -1,6 +1,20 @@
 #!/bin/sh
 
-EDITOR=mcedit	# crontab uses this editor
+# Mender image update need this
+# remove/add this crontab jobs
+/usr/bin/crontab -u root -l |  grep -v "* /opt/influx/release_check.sh" |  /usr/bin/crontab -
+#
+# extract preserved files
+if test -f /data/mender/preserved-files.tgz; then
+    tar -xzf /data/mender/preserved-files.tgz -C /
+    mv /data/mender/preserved-files.tgz /data/mender/preserved-files_old.tgz
+fi
+#
+# Avoiding the sleep during OTA update
+INF_UPG=$(/usr/sbin/fw_printenv influx_upgrade | cut -d = -f 2)
+if [ $INF_UPG == "1" ]; then
+        /usr/bin/systemctl start end_influx_upgrade.service
+fi
 
 # Power up the Quectel chip
 /opt/influx/cellular_module_start.sh

 
